folium/utilities.py

```diff
@@ -12,62 +12,105 @@
 import copy
 import uuid
 import collections
-try:
-    from collections import abc
-except ImportError:
-    import collections as abc
 
 import numpy as np
+try:
+    import pandas as pd
+except ImportError:
+    pd = None
 
-from six import binary_type, text_type
+from six import binary_type, text_type, string_types
 from six.moves.urllib.parse import urlparse, uses_netloc, uses_params, uses_relative
 
 
 _VALID_URLS = set(uses_relative + uses_netloc + uses_params)
 _VALID_URLS.discard('')
 
 
-def _is_sized_iterable(arg):
-    """Validates the arg is Sized & Iterable"""
-    return isinstance(arg, abc.Sized) & isinstance(arg, abc.Iterable)
```

Comment:
Don't you think this was simpler with the `abc`? I'm fine with this approach but I could not see why the change at a first pass.

Reply:
Hmm I tried to change it into abc.Sequence, because it should also have `__getitem__`, but then it won't work for Numpy arrays. I don't know what other data types have a length but don't inherit from `abc.Sized` for example. I guess I prefer the try/except version because it's more explicit and handles duck types better than abc.